src/QMCApp/QMCMain.cpp

```diff
@@ -82,10 +83,15 @@ QMCMain::QMCMain(Communicate* c)
   qmc_common.print_git_info_if_present(app_summary());
   app_summary() << "=====================================================\n";
   qmc_common.print_options(app_log());
-  app_summary() << "\n  MPI Ranks             = " << OHMMS::Controller->size()
-                << "\n  MPI Ranks per group   = " << myComm->size()
-                << "\n  MPI Group ID          = " << myComm->getGroupID()
-                << "\n  MPI Ranks per node    = " << NodeComm.size() << std::endl;
+  app_summary()
+      << "\n  MPI Ranks in total    = " << OHMMS::Controller->size()
+      << "\n  Number of rank groups = " << myComm->getNumGroups()
+      << "\n  MPI Group ID          = " << myComm->getGroupID()
+      << "\n  MPI Ranks in group    = " << myComm->size()
+      << "\n  MPI Ranks per node    = " << NodeComm.size()
+      << std::endl;
+  // assign accelerators within a node
```

Comment:
Add a note for QMC_MPI=0 builds?

Reply:
Now prints "This binary is built without MPI"